content/en/apps/guides/hosting/monitoring/integration.md

```diff
@@ -0,0 +1,204 @@
+
+Starting with the `Caddyfile`, let's assume your server's DNS entry is `cht.example.com`.  We can expose cAdvisor's service running on localhost port `8443` with this compose file. This tells Caddy to reverse proxy requests to the public interface to the private Docker network interface on port `8080` where cAdvisor is running:
+
+```yaml
+cht.example.com:8443 {
+    reverse_proxy cadvisor:8080
+}
+```
+
+Then we can add the compose file to run Caddy. Note that it's mounting the config file we just created:
+
+```yaml
+version: "3.9"
+services:
+  caddy:
+    image: caddy:2-alpine
+    restart: unless-stopped
+    ports:
+      - "8443:8443"
+    volumes:
+      - /home/ubuntu/Caddyfile:/etc/caddy/Caddyfile
+    networks:
+      - cht-net
+```
+
+#### Start cAdvisor, Caddy and CHT Core with Docker
+
+Now that we have all the config files in place, you need to have Docker start everything together. This is so that the containers can see each other on the same `CHT Net` Docker network.  You will need to specify each of the compose files every time you start, stop or restart CHT instance so all the service stay running and connected.
+
+Assuming you followed the [production steps]({{< relref "apps/guides/hosting/4.x/self-hosting-single-node" >}}) to install the CHT, you use this Compose call to first stop all containers and then start them all up, including the new services:
+
+```shell
+cd /home/ubuntu/cht/upgrade-service
+docker stop $(docker ps --quiet)
+docker compose up --detach
+```
+
+Note that the CHT Upgrade Service will process all Docker Compose file in the `/home/ubuntu/cht/compose` directory for us and we don't need to explicity specify them in the `docker compose up` command.
+
+### On CHT Watchdog: Import Grafana Dashboard
+
+Now that cAdvisor is running on your CHT Core instance and CHT Watchdog's Prometheus has additional scrape configs to ingest the cAdvisor metrics, we can now visualize it in a Grafana Dashboard and then alert on it. 
+
+1. Log into your Watchdog instance
+2. Click the upper left hamburger menu and click "Dashboards"
+3. Find the "New" button on the left, click and choose "Import" from the drop down
+4. On the next page, scroll down to find "Import via grafana.com", enter ID `193` and click "Load"
+5. Confirm the "Name" and "Folder" values and select "Prometheus" as the data source in the dropdown. Finally, click the "Import" button at the bottom of the page.
+
+That's it! After following these steps, you should be looking at the cAdvisor dashboard [as shown above](#steps-to-new-integrations).  From here, you can both customize this dashboard as well as add alerts as needed.
+
+<!-- there wasn't enough space after the diagrams, adding CSS to increase padding -->
+<style>
+.mermaid { padding-bottom: 20px; }
```

Comment:
Was this something specific to this page, or should we try to add this globally?

Reply:
I'll push this live now and see if it makes sense to move this globally in another PR - thanks!

Reply:
global is a good idea!  fixing in #1083